src/test_s2c_srv.c

```diff
@@ -382,6 +382,9 @@ int test_s2c(Connection *ctl, tcp_stat_agent *agent, TestOptions *testOptions,
           log_println(0, "S2C test error: can't create pipe.");
         } else {
           if ((s2c_childpid = fork()) == 0) {
+            // Don't capture more than 12 seconds of packet traces:
+            //   10 second test + 2 seconds of slop
+            alarm(12);
```

Comment:
The two `alarm`s in this PR are implicitly sharing the constants 10 and 2. Can we make these named constants so both use the same value? Also, @collina  is saying he believes the right number is 15. It would be nice if we can make a single constant that the test for length and for how the alarm derives its total time.